tox.ini

```diff
@@ -1,15 +1,9 @@
 [tox]
 envlist = py26,py27,py33,py34
 
```

Comment:
The whole file looks much cleaner. Great. I'm giving a :shipit:
Aesthetically, do we also want to remove this blank line?